include/psa/crypto_config.h

```diff
@@ -1057,18 +1057,17 @@
 //#define MBEDTLS_ENTROPY_FORCE_SHA256
 
 /**
- * \def MBEDTLS_ENTROPY_HARDWARE_ALT
+ * \def MBEDTLS_PLATFORM_GET_ENTROPY_ALT
  *
- * Uncomment this macro to let Mbed TLS use your own implementation of a
- * hardware entropy collector.
- *
- * Your function must be called \c mbedtls_hardware_poll(), have the same
- * prototype as declared in library/entropy_poll.h, and accept NULL as first
- * argument.
- *
- * Uncomment to use your own hardware entropy collector.
+ * By default Mbed TLS uses platform-specific sources such as getrandom(),
+ * /dev/urandom or BCryptGenRandom() to gather entropy data. If these functions
+ * are not available for some reason (ex: working on a baremetal project), the
+ * following symbol allows the user to define a custom callback function named
+ * `mbedtls_platform_get_entropy()` that Mbed TLS will use to gather entropy
```

Comment:
To make the function name into a link:
```suggestion
 * mbedtls_platform_get_entropy() that Mbed TLS will use to gather entropy
```